exercises/concept/arcade-high-score/.meta/design.md

```diff
@@ -0,0 +1,35 @@
+# Design
+
+## Learning objectives
+
+- Know what a map is.
+- Know how to define a map literal.
+- Know how to add key/value pairs in a map.
+- Know how to get values from a map.
+- Know how to get keys from a map.
+- Know how to update values in a map.
+- Know how to remove items from a map.
+- Know how to retrieve all items in a map.
+- Know what a `HashMap` is.
+- Know what a `LinkedHashMap` is.
+- Know what a `TreeMap` is.
+- Know the main differences in `HashMap`, `LinkedHashMap` and `TreeMap`.
+
+## Out of scope
+
+- `AbstractMap`
+- `IdentityHashMap`
+- `WeakHashMap`
+- `EnumMap`
+- `ConcurrentHashMap`
+- `ConcurrentSkipListMap`
+- `SortedMap`
+
+## Concepts
+
+- `maps`
+
+## Prerequisites
+
+- `strings`: know how to use string formatting.
```

Comment:
Technically I don't think string formatting is used in this exercise, so maybe `strings` doesn't have to be a prerequisite.